extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -1,92 +1,77 @@
 <div class="wds-global-navigation">
 	<div class="wds-global-navigation__content-bar">
-		<a href="#" class="wds-global-navigation__content-item wds-global-navigation__content-item-logo">
+		<a href="<?= Sanitizer::encodeAttribute( $model['logo']['header']['href'] ); ?>" class="wds-global-navigation__logo">
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)